place_orders.py

```diff
@@ -224,22 +124,7 @@ def pre_signing_thread(client, ledger_client, market, subaccount, orders, lock):
 async def listen_to_block_stream_and_place_orders(batch_writer):
```

Comment:
For the call to 

```python
                    asyncio.create_task(
                        place_orders(
                            ledger_client,
                            current_block,
                            orders[current_block],
                            batch_writer,
                        )
                    )
```

in this function and in the other place_*.py scripts, this gives me a linter warning: "Coroutine 'create_task' is not awaited"